Tart/Tart.munki.recipe

```diff
@@ -46,11 +44,38 @@
     <string>com.github.wegotoeleven-recipes.pkg.Tart</string>
     <key>Process</key>
     <array>
+        <dict>
+            <key>Arguments</key>
+            <dict>
+                <key>find</key>
+                <string>amd64</string>
+                <key>input_string</key>
+                <string>%ARCH%</string>
+                <key>replace</key>
+                <string>x86_64</string>
+            </dict>
+            <key>Processor</key>
+            <string>com.github.homebysix.FindAndReplace/FindAndReplace</string>
```

Comment:
hmmm so at face value I don't agree with this change.  This adds a seemingly unnecessary dependency.  Can you explain why this custom processor is necessary?

Reply:
It is because the Intel download is available as `amd64` but for Munki's supported architectures you need to specify `x86_64`.

This Find and Replace processor allows you to configure only a single `ARCH` variable and have it cover both use cases.

The alternative is to have both `DOWNLOAD_ARCH` and `SUPPORTED_ARCH` variables that need to be configured as distinct values.